include/libdnf/rpm/package.hpp

```diff
@@ -38,280 +38,435 @@ class Goal;
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_installtime(DnfPackage * pkg)
+    /// @return The unix timestamp (seconds since epoch) when a package was installed.
+    ///         Return 0 if the package is not installed.
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.installtime
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_installtime(DnfPackage * pkg)
     unsigned long long get_install_time() const;
 
-    /// @brief Media number for the package
-    ///
-    /// @replaces dnf:dnf/package.py:attribute:Package.medianr
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_medianr(DnfPackage * pkg)
-    unsigned long long get_media_number() const;
-
-    /// @brief The rpmdb ID for the package
-    ///
-    /// @replaces dnf:dnf/package.py:attribute:Package.rpmdbid
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_rpmdbid(DnfPackage * pkg)
+    /// @return The rpmdb database id (primary key) of the installed RPM package
+    /// @since 5.0
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.rpmdbid
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_rpmdbid(DnfPackage * pkg)
     unsigned long long get_rpmdbid() const;
 
-    /// TODO get_repo - requires Repo
-    /// @replaces dnf:dnf/package.py:attribute:Package.repo
-    /// @replaces dnf:dnf/package.py:attribute:Package.repoid
-    /// @replaces dnf:dnf/package.py:attribute:Package.reponame
-    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_get_repo(DnfPackage * pkg)
-    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_reponame(DnfPackage * pkg)
+    /// @return Repo object that represents the repository the package is part of.
+    /// @since 5.0
+    /// @note This **doesn't** describe the repository the package was installed from.
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.repo
+    // @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_get_repo(DnfPackage * pkg)
     Repo * get_repo() const;
 
-    std::string get_repo_id() const;
+    /// @return Repoid of the repository the package is part of.
+    /// @since 5.0
+    /// @note This **doesn't** describe the repository the package was installed from.
+    //
+    // @replaces dnf:dnf/package.py:attribute:Package.repoid
+    // @replaces dnf:dnf/package.py:attribute:Package.reponame
+    // @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_reponame(DnfPackage * pkg)
+    std::string get_repoid() const;
+
     // TODO(dmach): getBugUrl() not possible due to lack of support in libsolv and metadata?
 
 protected:
-    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_new(DnfSack *sack, Id id)
+    // @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_new(DnfSack *sack, Id id)
     Package(SolvSack * sack, PackageId id);
+
+    /// @return RPM package Name as a cstring
```

Comment:
Is `cstring` a conventional name for the type? I know it's the name of a header... but haven't seen it used like this yet. I'd personally say:
> ... as a `const char *`

(note the backticks)

Though these are private so not of much importance.

Reply:
Fixed. Yes, they are private hence not rendered in the docs, but it still makes sense to keep them in a good shape.